### PR TITLE
MINOR: system test clean up

### PR DESCRIPTION
**Pull request by bbejeck**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @guozhangwang and @ableegoldman for review

**Comment by bbejeck** [comment]
kicked off branch builder https://jenkins.confluent.io/job/system-test-kafka-branch-builder/3193/

File: streams/upgrade-system-tests-0100/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeToCooperativeRebalanceTest.java [hunk 1]
```diff
@@ -31,14 +31,13 @@
 
     @SuppressWarnings("unchecked")
     public static void main(final String[] args) throws Exception {
-        if (args.length < 3) {
-            System.err.println("StreamsUpgradeTest requires three argument (kafka-url, zookeeper-url, properties-file) but only " + args.length + " provided: "
-                + (args.length > 0 ? args[0] + " " : "")
-                + (args.length > 1 ? args[1] : ""));
+        if (args.length < 2) {
+            System.err.println("StreamsUpgradeToCooperativeRebalanceTest requires two arguments (zookeeper-url, properties-file) but only " + args.length + " provided: "
+                + (args.length > 0 ? args[0] + " " : ""));
```
Comment (ableegoldman) [inline streams/upgrade-system-tests-0100/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeToCooperativeRebalanceTest.java]: Just wondering, why the extra space after `args[0]`?

**Review by ableegoldman — APPROVED** [review]
Thanks for the cleanup!

File: streams/upgrade-system-tests-0100/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeToCooperativeRebalanceTest.java (hunk 1 above)
Comment (bbejeck) [reply streams/upgrade-system-tests-0100/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeToCooperativeRebalanceTest.java]: Didn't notice that before, I'll clean it up.

**Comment by bbejeck** [comment]
system test passed - http://confluent-kafka-branch-builder-system-test-results.s3-us-west-2.amazonaws.com/2019-10-18--001.1571363407--bbejeck--MINOR_system_test_follow_up--2d4824a/report.html

**Comment by bbejeck** [comment]
updated this

**Review by guozhangwang — COMMENTED** [review]
Thanks for the cleanup @bbejeck , wondering if we can do the same cleanup for StreamsUpgradeTest?

Otherwise, LGTM

**Comment by bbejeck** [comment]
>wondering if we can do the same cleanup for StreamsUpgradeTest?

@guozhangwang ack, I'll do a follow-on PR for this.

**Pull request merged**